<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="IDnumber">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>I.D. Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="StudentProfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 2px solid #00674F;
    border-radius: 10px;
    padding: 1px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    color: black;</string>
   </property>
   <property name="text">
    <string>Student Profile</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MiddleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Middle Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelStudent">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #00674F;
    color: white;
    border: 2px solid #000000;
    border-radius: 5px;
     padding: 1px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    color: white; 
}

/ Hover effect for the button */
QPushButton:hover {
    background-color: #944E63;
    border: 2px solid #944E63;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="YearLevel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Year Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="LastName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Middlenameline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddStudent">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #00674F;
    color: white;
    border: 2px solid #000000;
    border-radius: 5px;
     padding: 1px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    color: white; 
}

/ Hover effect for the button */
QPushButton:hover {
    background-color: #944E63;
    border: 2px solid #944E63;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Yearlevelline">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="IDline">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
   </property>
   <property name="placeholderText">
    <string>e.g. 2000-0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="Gender">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="CourseCode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Course Code</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Genderline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
   </property>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Firstnameline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Lastnameline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CourseComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
